replication_tunable_consistency.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -e
+
+function wait_weaviate() {
+  echo "Wait for Weaviate to be ready"
+  for _ in {1..120}; do
+    if curl -sf -o /dev/null localhost:$1; then
```

Comment:
Maybe `if curl -sf -o /dev/null localhost:$1/v1/.well-known/ready` would be more appropriate to check for readiness

Reply:
Agreed, changed accordingly